### PR TITLE
remove defunct stageable_discovery config example

### LINKED ISSUE #243: was this TEMPLATE.yml 'use_container' comment change erroneous?
the second subdirectory was removed from the structure described in the comments:
https://github.com/sul-dlss/pre-assembly/pull/232/files#diff-2f847418c6070e9dcfbc34cb3fbc0a55L96

spawned by https://github.com/sul-dlss/pre-assembly/pull/232/files#r216766266

### PR DESCRIPTION
**Pull request by jmartin-sul**
see also #274

fixes #243

### CODE REVIEW

**Comment by jmartin-sul** [comment]
rebased on current master, force pushed updated branch

**Comment by atz** [comment]
I like the deletion part, not sure about the commentary

**Comment by jmartin-sul** [comment]
the comment seemed like an easy way to give some historical context on what would otherwise seem like an especially weird and useless bit of indirection.

imagining someone coming to this project fresh, i thought it'd be more helpful to indicate that the unnecessary indirection and unconfigurability of `stageable_discovery` is an artifact of partially finished refactoring, as opposed to something intentionally implemented that might be useful as-is.

of course, that could be figured out via a bit of archaeology on the repo.  but it seemed helpful to save future programmers many minutes of "why's this designed this way?  let me see what this issue 274 is about... oh this is probably cruft from v3-legacy, ok."

wouldn't fight hard for it, and could see shrinking it, but i think i'd at least want to include that explanation on the github issue, and still point to the issue in a comment by the code.

**Comment by atz** [comment]
To me the comment (itself) seems to suggest stageable_discovery is more of an intended but unimplemented feature, so it may not reflect your intent.  #274 is also a bit hard to follow with all the screenshots of chats and editors.  

If the point is to say `stageable_discovery` is deprecated, I'd hit it with a `@deprecated` comment.  Or I would go ahead and outright remove the related code now, so there isn't even anything to deprecate or comment about: that's what I was referring to by "Can we make a call on it now?"

**Comment by jmartin-sul** [comment]
we felt we didn't have time to excise the code completely, since today is the end of the work cycle, but i've touched up my comments in a way that hopefully addresses @atz's concerns.  rebased on master and re-pushed.
